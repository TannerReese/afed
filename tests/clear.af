# Test clearing of substitutions
y: 10, x: 56, z: 17,
x * y - z = `jbfdnfnn`,
x / y = `28 / 5`,
z - x = ``

