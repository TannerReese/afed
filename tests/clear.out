# Test clearing of substitutions
y: 10, x: 56, z: 17,
x * y - z = ``,
x / y = ``,
z - x = ``

